app/index.html

```diff
@@ -95,13 +96,36 @@
   <script src="bower_components/lodash/lodash.js"></script>
   <!-- endbower -->
   <!-- endbuild -->
-
+  <!-- build:js(.) scripts/ui.bootstrap.js -->
+  <script src="bower_components/ui.bootstrap/src/position/position.js"></script>
+  <script src="bower_components/ui.bootstrap/src/stackedMap/stackedMap.js"></script>
+  <script src="bower_components/ui.bootstrap/src/dateparser/dateparser.js"></script>
+  <script src="bower_components/ui.bootstrap/src/datepicker/datepicker.js"></script>
+  <script src="bower_components/ui.bootstrap/src/collapse/collapse.js"></script>
+  <script src="bower_components/ui.bootstrap/src/tooltip/tooltip.js"></script>
+  <script src="bower_components/ui.bootstrap/src/accordion/accordion.js"></script>
+  <script src="bower_components/ui.bootstrap/src/popover/popover.js"></script>
+  <script src="bower_components/ui.bootstrap/src/typeahead/typeahead.js"></script>
+  <script src="bower_components/ui.bootstrap/src/tabs/tabs.js"></script>
+  <script src="bower_components/ui.bootstrap/src/timepicker/timepicker.js"></script>
+  <script src="bower_components/ui.bootstrap/src/modal/modal.js"></script>
+  <script src="bower_components/ui.bootstrap/src/alert/alert.js"></script>
+  <script src="bower_components/ui.bootstrap/src/buttons/buttons.js"></script>
+  <script src="bower_components/ui.bootstrap/src/carousel/carousel.js"></script>
+  <script src="bower_components/ui.bootstrap/src/pagination/pagination.js"></script>
+  <script src="bower_components/ui.bootstrap/src/progressbar/progressbar.js"></script>
+  <script src="bower_components/ui.bootstrap/src/rating/rating.js"></script>
```

Comment:
@kpunjani @rassiel  I have loaded all of the JS files for [UI-Bootstrap](https://angular-ui.github.io/bootstrap/). Please review the list of controls to become familiar with what is available to us. Also note (elsewhere) that I'm using [Angular UI Grid](http://ui-grid.info/) for grid management. In general I'm trying to take advantage of the fully-Angular projects (without JQuery) in the [Angular UI](https://angular-ui.github.io/) suite.